README.md

```diff
@@ -1,15 +1,20 @@
+# Hello-microverse
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
```

Comment:
- [ ] you did great using the right readme template , kindly remove `# Project Name` line since you already mentioned the project name